hayeon/programmers-1/나머지가-1이-되는-수-찾기.js

```diff
@@ -0,0 +1,24 @@
+function solution(n) {
+  for (i = 3; i <= n; i++) {
+    if (n % i === 1) return i;
+  }
+}
```

Comment:
```suggestion
function solution(n) {
  for (let i = 2; i <= n; i++) {
    if (n % i === 1) return i;
  }
}
```
- 잘푸셨습니다 고생하셨습니다 👍 윤우님 리뷰를 보시면 하연님께 드릴 피드백과 같은 피드백을 드렸습니다 참고부탁드립니다.
- 그리고 i를 3부터하면 풀이 실패가나옵니다 2부터 돌리셔야됩니다 🙆🏻‍♂️ 제출 전에 한번 풀이를 돌려보시길 바랄게요!

Reply:
넵 확인했습니당!!🤓